packages/image/src/providers/vercel.js

```diff
@@ -0,0 +1,18 @@
+import { set_param, relative_url } from './utils';
+
+// https://vercel.com/docs/concepts/image-optimization
```

Comment:
is there a page that shows what the possible query parameters for `/_vercel/image` are?